### ISSUE TITLE
Missing libplacebo dependency for moonlight-qt 6.0.0

### ISSUE THREAD

**Issue by SteenSchutt** [veldenb/plugin.program.moonlight-qt#70]
It seems like moonlight-qt introduced some sort of HDR support, which requires the presence of libplacebo v338.0 or later (Although probably v349 as that's what the AppImage seems to be looking for). Unfortunately this version is much too new to be found in the Debian repositories. I was trying to just add in the latest available version to the dockerfile, but even the Debian Multimedia repository only has v264.

I think the only real ways to proceed is to either build moonlight-qt with --no-hdr, figure out a way to get libplacebo into the container (probably the best solution) or install it in a completely different way. Everything works perfectly fine if I just hack in the URL to the moonlight-qt-5.0.1 appimage and run the install again.

```
Platform generic (x86_64) running libreelec 12.0 detected...
Loading LibreELEC profile for setting up environment...
Using custom libraries from /storage/.kodi/userdata/addon_data/plugin.program.moonlight-qt/moonlight-qt/lib...
Using Qt library from /storage/.kodi/userdata/addon_data/plugin.program.moonlight-qt/moonlight-qt/lib/qt5...
Running with window manager...
Using Kodi hooks for libreelec...
--- Starting Moonlight ---
./moonlight-qt: error while loading shared libraries: libplacebo.so.349: cannot open shared object file: No such file or directory
```

**Comment by moi952** [comment]
> Hi,
> 
> It looks like a few libraries need to be added to `/storage/.kodi/addons/plugin.program.moonlight-qt/resources/build/generic/create_standalone_moonlight_qt.sh`: libplacebo, libshaderc_shared and libvulkan.
> 
> The `APP_IMG_DEPENDENCIES` var needs to be modified so it looks like this:
> 
> ```
> APP_IMG_DEPENDENCIES="
>   bin/moonlight
>   lib/libav*
>   lib/libSDL*
>   lib/libssl.so*
>   lib/libcrypto.so*
>   lib/libplacebo.so*
>   lib/libshaderc_shared.so*
>   lib/libvulkan.so*
> "
> ```
> 
> After modifying the file and updating moonlight it works again for me. I need to add this to the package, but my main PC is currently broken (faulty PSU). So when my PC works again I can update the package and deploy a new release.

Perfect it work :)
Thanks

**Comment by veldenb** [comment]
Should be fixed now in release v0.4.4.
